specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+            },
+            "login": {
+              "$ref": "#/definitions/Login",
+              "description": "The configuration settings of the login flow of users using ContainerApp Authentication/Authorization."
+            },
+            "httpSettings": {
+              "$ref": "#/definitions/HttpSettings",
+              "description": "The configuration settings of the HTTP requests for authentication and authorization requests made against ContainerApp Authentication/Authorization."
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "GlobalValidation": {
+      "description": "The configuration settings that determines the validation flow of users using ContainerApp Authentication/Authorization.",
+      "type": "object",
+      "properties": {
+        "unauthenticatedClientAction": {
+          "description": "The action to take when an unauthenticated client attempts to access the app.",
+          "enum": [
+            "RedirectToLoginPage",
+            "AllowAnonymous",
+            "Return401",
+            "Return403"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "UnauthenticatedClientAction",
+            "modelAsString": true
+          }
+        },
+        "redirectToProvider": {
+          "description": "The default authentication provider to use when multiple providers are configured.\nThis setting is only needed if multiple providers are configured and the unauthenticated client\naction is set to \"RedirectToLoginPage\".",
+          "type": "string"
+        }
+      }
+    },
+    "Login": {
+      "description": "The configuration settings of the login flow of users using ContainerApp Authentication/Authorization.",
+      "type": "object",
+      "properties": {
+        "routes": {
+          "$ref": "#/definitions/LoginRoutes",
+          "description": "The routes that specify the endpoints used for login and logout requests."
+        },
+        "preserveUrlFragmentsForLogins": {
+          "description": "<code>true</code> if the fragments from the request are preserved after the login request is made; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "allowedExternalRedirectUrls": {
+          "description": "External URLs that can be redirected to as part of logging in or logging out of the app. Note that the query string part of the URL is ignored.\nThis is an advanced setting typically only needed by Windows Store application backends.\nNote that URLs within the current domain are always implicitly allowed.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "LoginRoutes": {
+      "description": "The routes that specify the endpoints used for login and logout requests.",
+      "type": "object",
+      "properties": {
+        "logoutEndpoint": {
```

Comment:
>logoutEndpoint [](http://example.com/codeflow?start=9&length=14)

This is used in a context that implies multiple routes, but is only a single object with a single string property. If it's multi, how are multiple routes encoded?

Reply:
Update name